docs/snippets/tracer/captureHTTP.ts

```diff
@@ -4,5 +4,5 @@ import axios from 'axios'; // (1)
 const tracer = new Tracer({ serviceName: 'serverlessAirline' });
 
 export const handler = async (event: unknown, context: Context): Promise<void> => {
-    await axios.get('https://httpbin.org/status/200');
+  await axios.get('https://httpbin.org/status/200');
```

Comment:
Let's add `axios` as a dev dependency under `docs/snippets/package.json`. I believe currently it's using the one from the main workspace